src/data_review_tool/pages/about.py

```diff
@@ -1,32 +1,37 @@
 import dash
 import sys
 import os
+from pages.config import *
 
 dash.register_page(__name__)
 
-from dash import Dash, dcc, html, Input, Output, callback
-import plotly.express as px
-
-
+from dash import dash, dcc, html, Input, Output, callback
+import os
 
-df = px.data.tips()
-days = df.day.unique()
+with open('src/data_review_tool/assets/about.md', 'r') as file:
```

Comment:
suggestion: Let's replace this with an `os.path` command so that it is compatible with all operating systems.